lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+let rateTable = {};
+
+exports.register = (defaultRate, routes) => {
+  routes = routes.filter((route) => {
```

Comment:
Instead of filtering first and then iterating on the routes, since you're not `map`ping, you could just have one `forEach` with the conditionals inside of the loop.